### PR TITLE
Add getter for known migrations to runner.

### LINKED ISSUE #117: Feature request: Get latest known migration / all known migrations
`Runner` struct has a `migrations` property, but it is not public. Would be nice to expose that, and maybe even some convenience methods:

  - `get_last_known_migration()`
  - `are_all_migrations_applied()`

### PR DESCRIPTION
**Pull request by jakajancar**
Fixes #117

### CODE REVIEW

File: refinery_core/src/runner.rs [hunk 1]
```diff
@@ -230,7 +230,12 @@ impl Runner {
         }
     }
 
-    /// set the target version up to which refinery should migrate, Latest migrates to the latest version available
+    /// Get the known migrations.
+    pub fn get_migrations(self) -> Vec<Migration> {
```
Comment (jxs) [inline refinery_core/src/runner.rs:234]: we don't want to consume the Runner, should be `&self`

File: refinery_core/src/runner.rs [hunk 2]
```diff
@@ -230,7 +230,12 @@ impl Runner {
         }
     }
 
-    /// set the target version up to which refinery should migrate, Latest migrates to the latest version available
+    /// Get the known migrations.
```
Comment (jxs) [inline refinery_core/src/runner.rs:233]:
```suggestion
    /// Get the gathered migrations.
```

File: refinery_core/src/runner.rs (hunk 1 above)
Comment (jakajancar) [reply refinery_core/src/runner.rs:234]: And I'm guessing


    pub fn get_migrations(&self) -> &Vec<Migration> {
        &self.migrations
    }

is better than

    pub fn get_migrations(&self) -> Vec<Migration> {
        self.migrations.clone()
    }

(new to Rust :))

Comment (jxs) [reply refinery_core/src/runner.rs]: yeah, no need to clone, let the caller of the function decide if they want to call clone

**Pull request merged**